html/statbrowser.js

```diff
@@ -15,7 +15,7 @@ if (!String.prototype.trim) {
 }
 
 // Status panel implementation ------------------------------------------------
-var status_tab_parts = ["Loading..."];
+var status_tab_parts = [["Loading...", ""]];
```

Comment:
what's going on in this file?

Reply:
It expects nested lists so this makes the stat panel show it loading